compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -179,6 +179,17 @@ impl<'local, 'interner> Interpreter<'local, 'interner> {
         }
     }
 
+    fn elaborate_item<T>(
```

Comment:
Since this only supports functions, would a name e.g. `elaborate_function_with_rebound_generics` or similar be more accurate? (I see there's already an `elaborate_function` function, but I'm not sure how it differs.)

Reply:
This supports any item, not just functions